ApollosProject/apollos-prototype
# Moves ContentCardConnected into the ui-connected package

### How do I test this PR?

- Open the App
- Click on connected cards and make sure app doesn't crash.

### What automated tests did you write?
Same coverage and tests moved over to package.
## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
